Cargo.toml

```diff
@@ -80,6 +80,267 @@ resolver = "2"
+opentelemetry-otlp = { git = "https://github.com/MaterializeInc/opentelemetry-rust.git" }
+ordered-float = "3.4.0"
+os_info = "3.5.1"
+parquet2 = { version = "0.17.1", default-features = false }
+paste = "1.0.11"
+phf = "0.11.1"
+phf_codegen = "0.11.1"
+pin-project = "1.0.12"
+polonius-the-crab = "0.3.1"
+postgres = { git = "https://github.com/MaterializeInc/rust-postgres" }
+postgres_array = { git = "https://github.com/MaterializeInc/rust-postgres-array" }
+postgres-openssl = { git = "https://github.com/MaterializeInc/rust-postgres" }
+postgres-protocol = { git = "https://github.com/MaterializeInc/rust-postgres" }
+pprof = "0.11.1"
+predicates = "2.1.4"
+proc-macro2 = "1.0.47"
+prometheus = { version = "0.13.3", default-features = false }
+proptest = { git = "https://github.com/MaterializeInc/proptest.git", default-features = false, features = ["std"] }
+proptest-derive = { git = "https://github.com/MaterializeInc/proptest.git" }
+prost = "0.11.9"
+prost-build = "0.11.9"
+prost-reflect = "0.9.2"
+protobuf-native = "0.2.1"
+protobuf-src = "1.1.0"
+quote = "1.0.23"
+rand = "0.8.5"
+rdkafka = { git = "https://github.com/MaterializeInc/rust-rdkafka.git" }
+rdkafka-sys = { git = "https://github.com/MaterializeInc/rust-rdkafka.git" }
+regex = "1.7.0"
+regex-syntax = "0.6.28"
+reqwest = "0.11.13"
+rlimit = "0.8.3"
+rocksdb = { version = "0.21.0", default-features = false }
+rpassword = "7.2.0"
+ryu = "1.0.12"
+schemars = "0.8.11"
+scopeguard = "1.1.0"
+security-framework = "2.7.0"
+segment = { version = "0.2.1", default-features = false }
+semver = "1.0.16"
+sentry = "0.29.1"
+sentry-tracing = "0.29.1"
+serde = "1.0.152"
+serde_json = "1.0.89"
+serde_plain = "1.0.1"
+serde_regex = "1.1.0"
+serde_urlencoded = "0.7.1"
+sha1 = "0.10.5"
+sha2 = "0.10.6"
+shell-words = "1.1.0"
+similar = "2.2.1"
+smallvec = "1.10.0"
+snap = "1.1.0"
+socket2 = "0.4.7"
+ssh-key = "0.4.3"
+stacker = "0.1.15"
+syn = "1.0.107"
+sysctl = "0.5.4"
+sysinfo = "0.27.2"
+tar = "0.4.38"
+tempfile = "3.3.0"
+termcolor = "1.1.3"
+thiserror = "1.0.37"
+tiberius = { version = "0.11.3", default-features = false }
+tikv-jemalloc-ctl = "0.5.0"
+tikv-jemallocator = "0.5.0"
+time = "0.3.17"
+timely = { git = "https://github.com/TimelyDataflow/timely-dataflow", default-features = false }
+tokio = "1.27.0"
+tokio-native-tls = "0.3.0"
+tokio-openssl = "0.6.3"
+tokio-postgres = { git = "https://github.com/MaterializeInc/rust-postgres" }
+tokio-stream = "0.1.11"
+tokio-test = "0.4.2"
+tokio-util = "0.7.4"
+toml = "0.5.9"
+tonic = "0.8.2"
+tonic-build = "0.8.2"
+tower = "0.4.13"
+tower-http = "0.3.5"
+tracing = "0.1.37"
+tracing-core = "0.1.30"
+tracing-opentelemetry = { git = "https://github.com/MaterializeInc/tracing.git", branch = "v0.1.x" }
+tracing-subscriber = "0.3.16"
+tungstenite = "0.18.0"
+typemap_rev = "0.3.0"
+uncased = "0.9.7"
+unicode-width = "0.1.10"
+url = "2.3.1"
+uuid = "1.2.2"
+walkdir = "2.3.2"
+yansi = "0.5.1"
+zeroize = "1.5.7"
+
+mz-adapter = { path = "src/adapter" }
```

Comment:
do we get any particular benefit from putting all the path deps here, too? if not, i'm inclined to say we should leave them inlined

Reply:
I only separated them because that's what we used to do with imports, I'll inline them!